src/olympia/devhub/tests/test_views.py

```diff
@@ -3835,3 +3835,29 @@ def test_devhub_version_edit_page(self):
     def test_devhub_version_list_page(self):
         url = reverse('devhub.addons.versions', args=[self.addon.slug])
         self.assertNameAndNoXSS(url)
+
+
+class TestSwitchToUnlisted(UploadTest):
```

Comment:
Those tests do not test this specific case where a file is pending review while the user changes to unlisted. Anyway, I'll move my test to `test_views_versions.py`.